inst/extdata/config.yaml

```diff
@@ -1,58 +1,40 @@
 # This is a config file for the piccr package
 
-FILE_ID: "*.csv"	# file extension used for data files
-COLUMNSEP: ","		# column separator used in data files
-DECSEP: "."   		# decimal separator used in data files
-NAME_StdScheme: IsoLabP_Muench2014.v2  # name of applied standard scheme (optional meta information)
-STD_NO:	5			  # number of standards used in standard scheme
-BLOCK_NO:	3			# number of standard blocks distributed over measurement rack in this standard scheme
-CALIBRATION_FLAG:	2			# set type of desired calibration: flag=1->drift-correction+three-point calibration/flag=2->double three-point calibration
-MEMORYCORR_FLAG:	1			# signal memory correction: 1=YES/0=NO
-OUTPUT_FLAG:	1			# format of desired csv-file output: flag=1->only output sample data/flag=2->output standard and sample data in original order
+file_extension: "*.csv"	          # file extension used for data files
+input_directory:  data            # directory with the measurement data to be calibrated
+output_directory: calibrated_data # output directory for the calibrated data
```

Comment:
`input_directory` und `output_directory` wurden hinzugefügt.

Reply:
That's a good idea. I would in addition suggest to include a flag for the number of injections to average over: `default` for averaging over all available injections, `num` = integer to average over the last `num` injections. What do you think?

Reply:
I think it's a good idea. Will add